### PR TITLE
feat(core, api-client): Serialize object arguments in desktop runtime

### PR DESCRIPTION
**Pull request by thisisamir98**
See https://github.com/wireapp/wire-webapp/pull/18359

### CODE REVIEW

File: packages/api-client/src/APIClient.ts [hunk 1]
```diff
@@ -200,7 +200,7 @@ export class APIClient extends EventEmitter {
   }
 
   private configureApis(backendFeatures: BackendFeatures): Apis {
-    this.logger.info('configuring APIs with config', backendFeatures);
+    this.logger.info('configuring APIs with config', JSON.stringify(backendFeatures));
```
Comment (olafsulich) [inline packages/api-client/src/APIClient.ts:203]: after changes in https://github.com/wireapp/wire-webapp/pull/18359, shouldn't it be done automatically?

Comment (thisisamir98) [reply packages/api-client/src/APIClient.ts:203]: web-packages use a separate instance of logdown of their own

Comment (thisisamir98) [reply packages/api-client/src/APIClient.ts:203]: see https://github.com/search?q=repo%3Awireapp%2Fwire-web-packages%20%3D%20logdown(&type=code

Comment (olafsulich) [reply packages/api-client/src/APIClient.ts:203]: okay, so maybe we should copy-paste the solution from the webapp you have provided? then we wouldn't have to stringify those args in the future, WDYT?

Comment (thisisamir98) [reply packages/api-client/src/APIClient.ts:203]: doing a big refactor

Comment (thisisamir98) [reply packages/api-client/src/APIClient.ts]: https://github.com/wireapp/wire-web-packages/pull/6696/commits/b25192005674d1fb61b74012850a9fc0fa9a4030

**Review by olafsulich — APPROVED** [review]

**Review by e-maad — APPROVED** [review]

**Pull request merged**